packages/amplify_api_plugin_interface/lib/amplify_api_graphql_helpers_interface.dart

```diff
@@ -0,0 +1,62 @@
+/*
+ * Copyright 2020 Amazon.com, Inc. or its affiliates. All Rights Reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License").
+ * You may not use this file except in compliance with the License.
+ * A copy of the License is located at
+ *
+ *  http://aws.amazon.com/apache2.0
+ *
+ * or in the "license" file accompanying this file. This file is distributed
+ * on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either
+ * express or implied. See the License for the specific language governing
+ * permissions and limitations under the License.
+ */
+
+// TODO: Datastore dependencies temporarily added in API. Eventually they should be moved to core or otherwise reconciled to avoid duplication.
+import 'package:amplify_datastore_plugin_interface/amplify_datastore_plugin_interface.dart';
+
+import 'src/types.dart';
+import 'src/GraphQL/GraphQLRequest.dart';
+import 'src/types/pagination/model_pagination.dart';
+import 'src/types/pagination/paginated_result.dart';
+
+abstract class ModelQueriesInterface {
+  // Get
+  static GraphQLRequest<T> get<T extends Model>(
+      ModelType modelType, String id) {
```

Comment:
Question: Can this be changed to be `ModelType<T> modelType, ...` to indicate the model type is for the same model used as generic for request?

Reply:
Yeah it can, good idea.